projects/plugins/jetpack/modules/masterbar/admin-menu/load.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Admin Menu loader.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Dashboard_Customizations;
+
+if ( jetpack_is_atomic_site() ) {
+	require_once __DIR__ . '/class-atomic-admin-menu.php';
+	Atomic_Admin_Menu::get_instance();
+} else {
```

Comment:
As this code also runs in wpcom, it seems we need also to check that this is not a Simple site also in order to load the Jetpack menu.

Reply:
38b7aba